src/types/balance.rs

```diff
@@ -15,41 +16,41 @@ pub struct Balance {
 impl Balance {
     /// All coinbase outputs not yet matured
     #[wasm_bindgen(getter)]
-    pub fn immature(&self) -> u64 {
-        self.balance.immature.to_sat()
+    pub fn immature(&self) -> Amount {
+        self.balance.immature.into()
```

Comment:
Those are safe since we are only wrapping a `BdkAmount` with our `Amount` type, right? (which seems to be a recurrent patterns for every types we're adding this PR)

Reply:
yes, 90% of the `bdk-wasm` is wrapping `bdk` types and implementing `From` so you can do `into()`.
If you want to know if `into()` is safe you can look at the `From` implementations. In this lib, all `From` are safe except the ones we discussed on `non-exhaustive`.